stdlib/3/_importlib_modulespec.pyi

```diff
@@ -26,6 +26,7 @@ if sys.version_info >= (3, 4):
 class ModuleType:
```

Comment:
Shouldn't be needed, since `__doc__` is defined on `object`, which is implicitly inherited.